app/controllers/api/skater_profiles_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::SkaterProfilesController < ApplicationController
+  def show
+    profile = SkaterProfile.find_by(skater_id: current_skater.id)
```

Comment:
Move this query into a private method to reduce duplication (it's called again in the update method)